w08h03/test/pgdp/datastructures/RandomTest.java

```diff
@@ -0,0 +1,117 @@
+package pgdp.datastructures;
+
+import org.junit.jupiter.api.RepeatedTest;
+
+import java.nio.charset.StandardCharsets;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Random;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RandomTest {
+    Random intTest = new Random(100);
+    Random doubleTest = new Random(100);
+    Random dateTest = new Random(100);
+    Random stringTest = new Random(100);
+    
+    int numberOfTests = 1000;

```

Comment:
```suggestion
    // If you want to increase the number of runs each test makes, just increase
    // this number.
    //
    // Please note that the test might be slow if you are using VS Code.
    // IntelliJ seems to be much faster.
    int numberOfTests = 1000;
```

Das wäre meine letzte Anmerkung, dass wir einen kurzen Kommentar schreiben und den Hinweis geben, dass VS Code etwas langsam sein könnte (so war zu mindestens bei mir gewesen), damit andere direkt Bescheid wissen.

Sobald wir das haben, können wir von meiner Seite aus mergen :)